sdk/src/metrics/meter.cc

```diff
@@ -234,10 +238,37 @@ std::unique_ptr<WritableMetricStorage> Meter::RegisterMetricStorage(
   return storages;
 }
 
+std::unique_ptr<AsyncWritableMetricStorage> Meter::RegisterAsyncMetricStorage(
+    InstrumentDescriptor &instrument_descriptor)
+{
+  auto view_registry = meter_context_->GetViewRegistry();
+  std::unique_ptr<AsyncWritableMetricStorage> storages(new AsyncMultiMetricStorage());
+  auto success = view_registry->FindViews(
```

Comment:
no need for `success`.

Reply:
Have added the check from success, and print warning if failure.